bust-a-move-framework/src/main/java/bustamove/player/Score.java

```diff
@@ -1,11 +1,3 @@
-/*
- * File: Score.java
- * Class: Score
```

Comment:
I remember removing it because checkstyle marked it as red. Turns out the error wasn't there. I'll put it back